internal/kokoro/check_incompat_changes.sh

```diff
@@ -6,13 +6,17 @@ set -x
 # Only run apidiff checks on go1.11 (we only need it once).
 # TODO(deklerk) We should pass an environment variable from kokoro to decide
 # this logic instead.
-if [[ `go version` != *"go1.11"* ]]; then
+if [[ `go version` != *"go1.12"* ]]; then
     exit 0
 fi
 
-try3() { eval "$*" || eval "$*" || eval "$*"; }
+if git log -1 | grep BREAKING_CHANGE_ACCEPTABLE; then
```

Comment:
Is there somewhere that documents the usage of this convention WRT contributing to this repo? A toggle for parts of CI should be documented in the README or CONTRIBUTING guidelines imo, even if just to inform the future maintainers

Reply:
There is now - I've added to the CONTRIBUTING.md in this CL.